DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/PresetServlet.java

```diff
@@ -42,7 +42,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    /*
     // parse each parameter from the save preset form.
     String userEmail = request.getParameter("userEmail");
```

Comment:
Why are you taking in user email?

Reply:
I don't remember how I was using it, I'll get rid of it.